addons/website_sale/static/tests/tours/website_sale_complete_flow.js

```diff
@@ -1,8 +1,6 @@
-/** @odoo-module **/
-
     import { rpc } from "@web/core/network/rpc";
```

Comment:
maybe worth investigating if we want to change whole file indentation here

Reply:
maybe in another commit/PR?

Reply:
Well, you reindent the two following lines, so it would make sense to at least reindent that one :eyes:

Reply:
OH just the imports? Yes indeed then, I though you wanted to reindent the whole file

Reply:
at least the imports, but the whole file would have worked too :p